### DISCUSSION TITLE
How does the NER model work?

### DISCUSSION THREAD

**Discussion by jaskaran-td** [explosion/spaCy#10513] [Help: Model Advice]
i was going through the spacy library code from there i am not able to understand the ner model architecture . As we have tok2vec,lower and upper in ner i want to know the layer architecture of every model as we get in tensorflow.

**Comment by polm** [comment]
The code is a little hard to follow, but the NER architecture is actually the same as the parser architecture. 

This article is relatively old now, but @honnibal  wrote about how the an early version of the parser works [here](https://explosion.ai/blog/parsing-english-in-python), which may help in understanding it.

**Reply by jaskaran-td** [reply]
@polm Thank you . let me go through this blog
